SonarQube.TeamBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -150,8 +150,10 @@ private bool FetchArgumentsAndRulesets(ProcessedArgs args, string configDir, ILo
 
                 // Generate the FxCop rulesets
                 logger.LogInfo(Resources.MSG_GeneratingRulesets);
-                GenerateFxCopRuleset(server, args.ProjectKey, "csharp", "cs", "fxcop", Path.Combine(configDir, FxCopCSharpRuleset), logger);
-                GenerateFxCopRuleset(server, args.ProjectKey, "vbnet", "vbnet", "fxcop-vbnet", Path.Combine(configDir, FxCopVBNetRuleset), logger);
+                GenerateFxCopRuleset(server, args.ProjectKey, "csharp", "cs", "fxcop", Path.Combine(settings.SonarConfigDirectory, FxCopCSharpRuleset), logger);
+                GenerateFxCopRuleset(server, args.ProjectKey, "vbnet", "vbnet", "fxcop-vbnet", Path.Combine(settings.SonarConfigDirectory, FxCopVBNetRuleset), logger);
+
+                SonarLintAnalyzerProvider.SetupAnalyzers(server, settings, args.ProjectKey, logger);
```

Comment:
I tried to keep the changes to the pre-processor to a minimum, so it just makes this single call to new analyser provider.